helm/dbp-keycloak/templates/ingress.yaml

```diff
@@ -1,27 +1,30 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ template "common.names.name" . }}
+  name: {{ template "common.names.fullname" . }}
   namespace: {{ template "common.names.namespace" . }}
-  labels: 
+  labels:
     {{- include "common.labels" . | nindent 4 }}
   {{- with .Values.ingress.annotations }}
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  {{- if .Values.ingress.tls.enabled }}
   tls:
     - hosts:
-        - {{ .Values.keycloakHostname  }}
+        - {{ .Values.keycloak.hostname }}
+      secretName: "{{ .Values.ingress.tls.secretName }}"
+  {{- end }}
   ingressClassName: {{ .Values.ingress.ingressClassName }}
   rules:
-    - host: {{ .Values.keycloakHostname  }}
+    - host: {{ .Values.keycloak.hostname }}
```

Comment:
More then one Host in one ingress definition can be solved via an rage over the host